### DISCUSSION TITLE
Is is possible to pass operator as variable/parameter in hasura?

### DISCUSSION THREAD

**Discussion by VSAmrutha** [hasura/graphql-engine#6122] [General]
Is is possible to pass operator as variable/parameter in hasura?
Like for one query the operator should be "_and" and for other query it should be "_or".

**Comment by praveenweb** [comment]
@VSAmrutha - You can template the whole `where` argument with a GraphQL Variable. For example:

```
query($where: users_bool_exp) {
  users(where: $where) {
    id
    name
  }
}
```

Now `$where` can have any value including `_and`, `_or` or any operator.

You can check this using the Docs tab inside GraphiQL interface to explore different input types.
